src/components/themes/__tests__/themes-test-cases.js

```diff
@@ -63,9 +70,10 @@ const customProps = {
   customTooltipText: 'Contact us for access to this feature.',
   customStyles: {
     background: '#FEDADA',
-    color: '#DA2E30',
+    color: '#c12325',
```

Comment:
This custom tag's contrast wasn't passing, so I updated it to make sure the Themes test cases app page has 100% on accessibility.